ts/util/table-builder.ts

```diff
@@ -137,22 +143,27 @@ export class TableBuilder<ParentType, ChildType> {
     const row = tbody.insertRow();
     const cell = row.insertCell();
     cell.colSpan = this.definition.columns.length;
-    const text = document.createTextNode("No data");
-    cell.appendChild(text);
+    cell.className =
+      "text-black p-4 bg-grey-100 font-bold border-grey-200 border-t-2";
+    cell.appendChild(document.createTextNode("NO DATA"));
   }
 
   private addParentCell(
     tr: HTMLTableRowElement,
     column: ColumnDefinition<ParentType, ChildType>,
     parent: ParentType,
-    children: ChildType[]
+    children: ChildType[],
+    index: number
   ) {
     if (!column.addParentContents) {
       throw new Error(
         `Column "${column.title}" specified as parent, but doesn't define addParentContents`
       );
     }
-    const td = tr.insertCell();
+    const td = addCell(tr);
+    if (index > 0) {
+      td.classList.add("border-l-2");
+    }
```

Comment:
Add `index` parameter to `addCell` and handle border in there instead, so it's consistent with `addColumnHeader`